tests/Microsoft.Maui.IntegrationTests/Microsoft.Maui.IntegrationTests.csproj

```diff
@@ -0,0 +1,23 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net7.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+    <IsPackable>false</IsPackable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.5.0" />
+    <PackageReference Include="NUnit" Version="3.13.3" />
+    <PackageReference Include="NUnit3TestAdapter" Version="4.3.1" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Compile Remove="TestResources\*" />
+    <EmbeddedResource Include="TestResources\*">
+      <LogicalName>%(FileName)%(Extension)</LogicalName>
+    </EmbeddedResource>
+  </ItemGroup>
+
+</Project>
```

Comment:
I've put this under a new `tests` folder at the root of the repo, but should this integration test project move under `src` somewhere instead?

Reply:
maybe `\src\Templates\tests` ?

Reply:
I'm hoping that this project can be more widely used in the future, if we add more tests to it that aren't only for templates then we probably want it in a less specific folder.

Reply:
I moved this under src/TestUtils.